generators/server/generator.mjs

```diff
@@ -25,12 +25,15 @@ export default class extends GeneratorBaseEntities {
   get [POST_WRITING_PRIORITY]() {
     return {
       async packageJson() {
+        this.editFile('package.json', content => content.replaceAll('./mvnw', 'mvnw'));
+
         this.packageJson.merge({
           scripts: {
             'native-e2e': 'concurrently -k -s first "npm run native-start" "npm run e2e:headless"',
-            'native-package': './mvnw package -Pnative,prod -DskipTests',
+            'native-package': 'mvnw package -Pnative,prod -DskipTests',
```

Comment:
This fails on Linux and macOS. 

```
> mvnw package -Pnative,prod -DskipTests "-B" "-ntp" "-Dnative-image-name=flickr2-Linux-v0.0.1-x86_64" "-Dnative-build-args=--verbose -J-Xmx10g"

sh: 1: mvnw: not found
```

I think it might be best to just use `mvn` and open an issue to add better Maven Wrapper support later. That way, I can publish blog post and we can move on.

Reply:
Works inside npm scripts.
Tests are passing at Linux and MacOS here. And windows succeed the build step.
```
npm install
npx mvnw package -Pnative,prod -DskipTests "-B" "-ntp" "-Dnative-image-name=flickr2-Linux-v0.0.1-x86_64" "-Dnative-build-args=--verbose -J-Xmx10g"
```

I think this is a good approach for CI.
I just think using mvn at npm scripts is not nice, I myself don’t have maven installed globally.
But it’s ok to me if this will help your blog post.

Reply:
I'll try again with the changes you just made to this PR. I'm also [trying to get Windows + Docker to work](https://github.com/mraible/jhipster7-flickr2/commit/1ed0f8f75534ff2e4cba36b4753ef394cddf184b).